test/node/node_test.cpp

```diff
@@ -211,6 +211,20 @@ TEST(NodeTest, StdVector) {
   EXPECT_EQ(primes, node["primes"].as<std::vector<int>>());
 }
 
+TEST(NodeTest, StdDeque) {
```

Comment:
Is this request still relevant? Can you tell me what the missing tests are if any?